pkg/service/mockrecord/mockrecord.go

```diff
@@ -21,13 +21,13 @@ import (
 var Emoji = "\U0001F430" + " Keploy:"
 
 type mockRecorder struct {
-	logger *zap.Logger
+	Logger *zap.Logger
```

Comment:
If it is not used by the external packages, this can be unexported.

Reply:
done